internal/cli/explain.go

```diff
@@ -95,6 +96,10 @@ func (c *explainCmd) run(ctx context.Context, args []string) error {
 	a.Incr("cmd.explain", cmdTags.AsMap())
 	defer a.Flush(time.Second)
 
+	if len(args) == 0 {
```

Comment:
this is a good fix! but we usually try to keep this consistent with what upstream kubectl is doing. in this case, i think it's missing a call to Validate() after the Complete() call a few lines below this. does that work?
https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/explain/explain.go#L141

Reply:
Yeah that's much better, thanks!